legal-api/report-templates/amalgamationApplication.html

```diff
@@ -1,13 +1,241 @@
+      <table class="header-table" role="presentation">
+        <tr class="no-page-break">
+          <td>
+            [[logo.html]]
+          </td>
+          <td>
+            <div class="report-type">AMALGAMATION APPLICATION</div>
+            <div class="report-type-desc">BC Limited Company - Business Corporations Act</div>
+          </td>
+        </tr>
+      </table>
+      <div class="business-name-header">
+        <label class="lbl-business-name">{{business.legalName}}</label>
+      </div>
+    </div>
+    [[common/businessDetails.html]]
+    <div class="no-page-break">
+      <div class="section-header mt-5 pt-4">AMALGAMATION APPLICATION</div>
+    </div>
+
+    <!-- Name Request [[amalgamation/nameRequest.html]]-->
+    <div class="no-page-break">
+      <table class="section-data nr-name-table mt-4" role="presentation">
+          <tr>
+              <td class="bold col-33">Name Reservation Number:</td>
+              <td>
+                  {% if not amalgamationApplication.nameRequest.nrNumber %}
+                  N/A
+                  {% else %}
+                  {{amalgamationApplication.nameRequest.nrNumber}}
+                  {% endif %}
+              </td>
+          </tr>
+          <tr>
+              <td class="bold pt-2">Name Reserved:</td>
+              <td class="pt-2">
+                  {% if not amalgamationApplication.nameRequest.nrNumber %}
+                  The company is to be incorporated with a name created by adding "B.C. LTD." after the incorporation number.
+                  {% else %}
+                  {{amalgamationApplication.nameRequest.legalName}}
+                  {% endif %}
+              </td>
+          </tr>
+      </table>
+    </div>
+
+    <!-- Effective Date [[amalgamation/effectiveDate.html]]-->
+    <div class="no-page-break">
+      <div class="separator mt-4"></div>
+      <div class="section-title mt-4">Amalgamation Effective Date</div>
+      <div class="section-data mt-4">
+          {% if header.isFutureEffective %}
+              {{header.effective_date_time}}
+          {% else %}
+              The incorporation is to take effect at the time that this application is filed with the Registrar.
+          {% endif %}
+      </div>
+    </div>
+
+    <!-- Approval Type [[amalgamation/approvalType.html]]-->
+    <div class="no-page-break">
+      <div class="separator mt-4"></div>
+      <div class="section-title mt-4">
+        <span>Approval Type</span>
+      </div>
+      {% if amalgamationApplication.courtApproval is none %}
+        <div class="section-data mt-3">
+          N/A
+        </div>
+      {% else %}
+        {% if amalgamationApplication.courtApproval == 'courtOrder' %}
+          <div class="section-data mt-3">
+            The restoration was approved by a court order.
+          </div>
+          <div class="section-data mt-2">
+            Court order number: <span class="bold">{{amalgamationApplication.courtApproval.fileNumber}}</span>
```

Comment:
amalgamationApplication.courtOrder.fileNumber

Reply:
removed the file number completely, as not in the UX design